services/QuillLMS/app/services/serialize_evidence_activity_health.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class SerializeEvidenceActivityHealth
+
+  def initialize(activity)
+    @activity = activity
+  end
+
+  def data
+    {
+      name: @activity.title,
+      flag: @activity.flag.to_s,
+      activity_id: @activity.id,
+      version: @activity.version,
+      version_plays: FeedbackHistory.get_total_count({activity_id: @activity.id, activity_version: @activity.version}),
+      total_plays: FeedbackHistory.get_total_count({activity_id: @activity.id}),
+      completion_rate: activity_feedback_history[:average_completion_rate],
+      because_final_optimal: percent_final_optimal_for_conjunction(FeedbackHistory::BECAUSE),
+      but_final_optimal: percent_final_optimal_for_conjunction(FeedbackHistory::BUT),
+      so_final_optimal: percent_final_optimal_for_conjunction(FeedbackHistory::SO),
+      avg_completion_time: activity_feedback_history[:average_time_spent] ? Utils::Numeric.human_readable_time_to_seconds(activity_feedback_history[:average_time_spent]) : nil,
+    }
+  end
+
+  def prompt_data
+    # TODO: fill this in to return serialized prompt health objects
+    []
```

Comment:
You left a TODO here.  I assume that this is intended to be part of some follow-on PR?  Might be worth deleting this function in the interim if that might be a little bit out.

Reply:
Yeah, it's a note for my next PR. I'll leave it in for now since it should be done in a week or so.